studio/components/interfaces/Auth/AutoSchemaForm.tsx

```diff
@@ -13,25 +13,30 @@ import {
   FormSectionContent,
   FormSectionLabel,
 } from 'components/ui/Forms'
+import { useAuthConfigQuery } from 'data/auth/auth-config-query'
+import { useAuthConfigUpdateMutation } from 'data/auth/auth-config-update-mutation'
+import { useCheckPermissions, useStore } from 'hooks'
 
 const AutoSchemaForm = observer(() => {
-  const { authConfig, ui } = useStore()
-  const { isLoaded } = authConfig
+  const { ui } = useStore()
+  const { ref: projectRef } = useParams()
+  const { data: config, isFetched: isLoaded } = useAuthConfigQuery({ projectRef })
```

Comment:
same as ProvidersForm, lets use isSuccess, and also account for isLoading, isError

Reply:
Done.